onnxruntime/contrib_ops/cuda/custom_layer_norm.cu

```diff
@@ -0,0 +1,305 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/cuda/cu_inc/common.cuh"
```

Comment:
nit: in general, order include files per coding standards, use paths relative to base source directory
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes